build/find-missing-license.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
```

Comment:
Previously we've been using underscores for filenames. Is there something that led you to use dashes?

Reply:
matching make commands, less keystrokes